### ISSUE TITLE
Changes in the paths of the tar.gz packages of Nexus projects

### ISSUE THREAD

**Issue by p4535992** [Alfresco/alfresco-transform-core#978]
Today the machines were not starting for me and I verified an error on downloading packages from https://nexus.alfresco.com/nexus/.

I blandly looked at it and noticed that some urls were changed below are the ones I noticed for now:

`https://nexus.alfresco.com/nexus/service/local/repositories/thirdparty/content/org/libreoffice/libreoffice-dist/7.2.5/libreoffice-dist-7.2.5-linux.gz
`

is become

`https://nexus.alfresco.com/nexus/service/local/repositories/thirdparty/org/libreoffice/libreoffice-dist/7.2.5/libreoffice-dist-7.2.5-linux.gz`

and 

`https://nexus.alfresco.com/nexus/service/local/repositories/releases/content/org/alfresco/alfresco-pdf-renderer/1.2/alfresco-pdf-renderer-1.2-linux.tgz`

is become

`https://nexus.alfresco.com/nexus/service/local/repositories/releases/org/alfresco/alfresco-pdf-renderer/1.2/alfresco-pdf-renderer-1.2-linux.tgz`

basically it looks like they removed the “/content” portion from the path, I don't know if it's an error or if it's a new structure decided by Alfresco, but I point it this out.

**Comment by montgolfiere** [comment]
Thank you for pointing this out, including the new urls. Yes, apologies, it was fallout after Nexus was upgraded.
